CHANGELOG.md

```diff
@@ -2,6 +2,60 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [240.0.49](https://github.com/forcedotcom/commerce-on-lightning/compare/v238.0.16...v240.0.49) (2022-10-10)
```

Comment:
@joeluong-sfcc why do we still have changelog changes? I thought you already merged the changes from main to develop

Reply:
I'm not sure... This is the PR that merged in the changelog changes. It might be because I messed with one of the commits to sign it? But these changes already exist in develop I believe....